src/quote/QuoteSchema.ts

```diff
@@ -0,0 +1,74 @@
+const QuoteSchema = {
+  $schema: 'http://json-schema.org/draft-07/schema#',
```

Comment:
We will have to add a version here, too since it represents out spec

Reply:
Done.